demos/kdf/hkdf.c

```diff
@@ -95,6 +95,8 @@ int main(int argc, char **argv)
         goto end;
     }
 
+    printf("Success\n");
```

Comment:
Should the success message go to stdout?  or do we want it to go to stderr

Reply:
stdout seems reasonable to me.

Reply:
the general pattern I noticed in the demo code was that error messages go to stderr and status / success messages go to stdout, so I picked stdout above.

However, I just did `cd demos && make > stdout.txt` on my branch, and I see that `pkey/EVP_PKEY_RSA_keygen.c` does not follow that pattern (status messages go to stderr).  I will push an update for that.